### DISCUSSION TITLE
motion sensor - asleep (zzz)

### DISCUSSION THREAD

**Discussion by gkathan** [zwave-js/zwave-js-ui#3531] [General]
hi,
I have a fibaro motion sensor included via aeotec 7 stick ... it shows in the z-wave Js ui in home assistant as asleep (zzz) but works in recognizing motions BUT I am not able to change any configuration parameter ... it will just timeout when entering a new value ... I assume this is due to the sensors sleep state right? 
how can I somehow wake this thing up to transmit some config updates ?
any hint appreciated
cheerz from vienna![Screenshot_20240118-075706.png](https://github.com/zwave-js/zwave-js-ui/assets/6329814/a6a5120c-4a21-41ec-ad94-68ac71d8369f)

![Screenshot_20240118-075539.png](https://github.com/zwave-js/zwave-js-ui/assets/6329814/b73228a4-a0e5-4aca-b918-8aece2470267)

**Comment by kpine** [comment]
You'll have to check your device manual, it will tell you the wake up procedure, usually some kind of button press, often the same as inclusion/exclusion. Nearly all of Fibaro's manuals are online if you don't have one.

The settings are not timed out, the driver will queue them and send the commands when the device wakes up.

For example [this manual](https://manuals.fibaro.com/motion-sensor/) for a Fibaro motion sensor says:

> **Waking up the Motion Sensor:**
>
> FIBARO Motion Sensor needs to be woken up to receive information about the new configuration from the controller, like parameters and associations.
>
> To wake up the sensor manually triple click the B-button located inside the housing.

**Reply by kpine** [reply]
Also, some devices have built-in help that tells you how to wake them update, which is obtained from the [device DB](https://devices.zwave-js.io/?jumpTo=0x0138:0x0001:0x0003:0.0).

![image](https://github.com/zwave-js/zwave-js-ui/assets/791794/fa538a48-2222-419a-8442-6b99b955ab2d)

But if your motion sensor is the [FGMS001](https://devices.zwave-js.io/?jumpTo=0x010f:0x0800:0x1001:0.0), it does not have that info.
